.github/workflows/lint_and_tests.yml

```diff
@@ -0,0 +1,32 @@
+name: lint_and_tests
```

Comment:
@Paulooh007 could you please merge the `main` branch into yours?
This way, we will get rid of all the "files changed" that don't have any relation to your changes in this PR.